blackbox/docs/sql/reference/set.txt

```diff
@@ -5,48 +5,60 @@
 SET / RESET
 ===========
 
-Alter and restore global cluster setting values at runtime.
-For a list of the possible settings see
-:ref:`conf-cluster-settings`. Only settings documented with
+Change and restore settings at runtime. To get an overview of possible Crate
+settings, see :ref:`conf-cluster-settings`. Only settings documented with
 *Runtime:*  ``yes`` can be changed.
 
-
 Synopsis
 ========
 
 ::
 
-    SET GLOBAL [ PERSISTENT | TRANSIENT ] setting_ident [ = | TO ] value [ , ... ]
+    SET [ SESSION | LOCAL ] setting_ident { = | TO } { value | ident | DEFAULT }
+
+    SET GLOBAL [ PERSISTENT | TRANSIENT ] setting_ident [ = | TO ] { value [, ...] | ident [, ...] }
```

Comment:
`SET GLOBAL [ PERSISTENT | TRANSIENT ] ( setting_ident [ = | TO ] { value | ident } ) [, ...]`
you know what I mean, right?